<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Дневник</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Оценки</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="button_add_grade">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Добавить оценку</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_change_grade">
            <property name="text">
             <string>Изменить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_delete_grade">
            <property name="text">
             <string>Удалить оценку</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_student_record">
            <property name="text">
             <string>Успеваемость ученика</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_class_record">
            <property name="text">
             <string>Успеваемость класса</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table_grades"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Ученики</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="rightMargin">
           <number>183</number>
          </property>
          <item>
           <widget class="QPushButton" name="button_add_student">
            <property name="text">
             <string>Добавить ученика</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_change_student">
            <property name="text">
             <string>Изменить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_delete_student">
            <property name="text">
             <string>Удалить ученика</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_find_students">
            <property name="text">
             <string>Найти учеников</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table_students"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Классы</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="rightMargin">
           <number>366</number>
          </property>
          <item>
           <widget class="QPushButton" name="button_add_class">
            <property name="text">
             <string>Добавить класс</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_change_class">
            <property name="text">
             <string>Изменить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_delete_class">
            <property name="text">
             <string>Удалить класс</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table_classes"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Предметы</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="rightMargin">
           <number>366</number>
          </property>
          <item>
           <widget class="QPushButton" name="button_add_subject">
            <property name="text">
             <string>Добавить предмет</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_change_subject">
            <property name="text">
             <string>Изменить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_delete_subject">
            <property name="text">
             <string>Удалить предмет</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table_subjects"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
